File: android/app/src/main/java/social/courier/app/model/Preferences.kt
package social.courier.app.model

data class Preferences(
    val likes: Boolean = true,
    val replies: Boolean = true,
    val reposts: Boolean = true,
    val follows: Boolean = true,
    val mentions: Boolean = true,
    val quotes: Boolean = true,
    val generic: Boolean = false
)

File: ios/Courier/Courier/Courier/Models/Preferences.swift
import Foundation

struct Preferences: Codable {
    var likes: Bool
    var replies: Bool
    var reposts: Bool
    var follows: Bool
    var mentions: Bool
    var quotes: Bool
    var generic: Bool

    static let `default` = Preferences(
        likes: true,
        replies: true,
        reposts: true,
        follows: true,
        mentions: true,
        quotes: true,
        generic: false
    )
}
